tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: Couchbase | Install Couchbase Repo
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - http://packages.couchbase.com/releases/couchbase-release/couchbase-release-1.0-0-x86_64.rpm
+    - couchbase-server-4.5.0
+
+- name: Couchbase | Create Data Directory
+  file:
+    path: /data
+    state: directory
+    owner: couchbase
+    group: couchbase
+    mode: 0755
+
+- name: Couchbase | Enable Startup
+  service:
+    name: couchbase-server
+    state: started
+    enabled: yes
+
+- name: Couchbase | Calculate Memory Overheads
+  set_fact:
+    cb_mem_size: "{{ ansible_memtotal_mb * 0.75}}"
```

Comment:
the 0.75 should also be configurable :)

Reply:
This has been removed in later commits in favour of explicitly setting values - I had trouble with calculations on the fly which stopped cluster setup. Definitely want to revisit this so it scales dep on available resources